packages/fxa-shared/metrics/glean/web/index.ts

```diff
@@ -51,6 +51,12 @@ export const eventsMap = {
     success: 'login_email_confirmation_success_view',
   },
 
+  loginRecoveryCode: {
```

Comment:
Could we potentially call this `loginBackupCode`? I know we currently use recovery codes in the auth-server, but it would be nice to disambiguate from the the recovery key if possible.

Reply:
I went with the component/page name in fxa-settings but I don't have strong feeling about it.